lte/gateway/python/magma/mobilityd/ip_allocator_static.py

```diff
@@ -18,190 +18,75 @@
 from __future__ import absolute_import, division, print_function, \
     unicode_literals
 
-import logging
-from collections import defaultdict
 from ipaddress import ip_address, ip_network
-from typing import List, Set
-
-from copy import deepcopy
-from magma.mobilityd.ip_descriptor import IPDesc, IPState
-from .ip_descriptor_map import IpDescriptorMap
-from .ip_allocator_base import IPAllocator, NoAvailableIPError, \
-    IPBlockNotFoundError, OverlappedIPBlocksError
+from typing import List
 
+from magma.mobilityd.ip_descriptor import IPDesc, IPState, IPType
+from magma.mobilityd.ip_allocator_base import IPAllocator
+from magma.mobilityd.subscriberdb_client import SubscriberDbClient
 
 DEFAULT_IP_RECYCLE_INTERVAL = 15
 
 
 class IpAllocatorStatic(IPAllocator):
 
-    def __init__(self,
-                 assigned_ip_blocks: Set[ip_network],
-                 ip_state_map: IpDescriptorMap,
-                 sid_ips_map: defaultdict):
-        """ Initializes a new IP allocator
+    def __init__(self, rpc_stub, ip_allocator: IPAllocator):
```

Comment:
rpc_stub sounds generic, can we update it to subscriberdb_rpc_stub? also it would be great updating the paremeter with typing (SubscriberDBStub)